src/OrbitGl/Batcher.cpp

```diff
@@ -44,11 +44,20 @@ void Batcher::AddVerticalLine(Vec2 pos, float size, float z, const Color& color,
   AddLine(pos, pos + Vec2(0, size), z, color, picking_color, nullptr);
 }
 
+static void MoveLineToPixelCenterIfHorizontal(Line& line) {
+  if (line.start_point[1] != line.end_point[1]) return;
+  line.start_point[1] += 0.5f;
+  line.end_point[1] += 0.5f;
+}
+
 void Batcher::AddLine(Vec2 from, Vec2 to, float z, const Color& color, const Color& picking_color,
                       std::unique_ptr<PickingUserData> user_data) {
   Line line;
+  // TODO(b/195386885) This is a hack to address the issue that some horizontal lines in the graph
```

Comment:
Please move the comment two lines below (above the hack)

Reply:
Done. Thanks!